davinci/src/androidTest/kotlin/com/pingidentity/davinci/FormFieldValidationTest.kt

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (c) 2025 Ping Identity. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package com.pingidentity.davinci
+
+import androidx.test.filters.SmallTest
+import com.pingidentity.davinci.collector.FlowCollector
+import com.pingidentity.davinci.collector.InvalidLength
+import com.pingidentity.davinci.collector.Length
+import com.pingidentity.davinci.collector.MinCharacters
+import com.pingidentity.davinci.collector.PasswordCollector
+import com.pingidentity.davinci.collector.RegexError
+import com.pingidentity.davinci.collector.Required
+import com.pingidentity.davinci.collector.TextCollector
+import com.pingidentity.davinci.collector.UniqueCharacter
+import com.pingidentity.davinci.module.Oidc
+import com.pingidentity.davinci.plugin.collectors
+import com.pingidentity.logger.Logger
+import com.pingidentity.logger.STANDARD
+import com.pingidentity.orchestrate.ContinueNode
+import com.pingidentity.testrail.TestRailCase
+import com.pingidentity.testrail.TestRailWatcher
+import junit.framework.TestCase.assertFalse
+import kotlinx.coroutines.test.runTest
+import org.junit.Rule
+import org.junit.rules.TestWatcher
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+@SmallTest
+class FormFieldValidationTest {
+    private var daVinci = DaVinci {
+        logger = Logger.STANDARD
+
+        module(Oidc) {
+            clientId = "60de77d5-dd2c-41ef-8c40-f8bb2381a359"
```

Comment:
Let's create a ticket enhance it later. Ideally those environment settings should come from another file, not committed to the repo.

Reply:
Good point, @rodrigoareis. 
I created [SDKS-3727](https://pingidentity.atlassian.net/browse/SDKS-3727) to address this issue.